pkgs/applications/misc/qjournalctl/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub, qtbase, qmake }:
+
+stdenv.mkDerivation rec {
+  pname = "qjournalctl";
+  version = "2018-11-25";
```

Comment:
https://nixos.org/nixpkgs/manual/
> If a package is not a release but a commit from a repository, then the version part of the name must be the date of that (fetched) commit. The date must be in "YYYY-MM-DD" format. Also append "unstable" to the name - e.g., "pkgname-unstable-2014-09-23".

Reply:
I hate adding `unstable`.  :imp: 

But that is *not* what this PR is about, and I'll add it shortly :).  Thanks!